test/blackbox-tests/test-cases/reason/run.t

```diff
@@ -1,94 +1,109 @@
   $ dune build @runtest @install-file --display short
+  Info: creating file dune-project with this contents: (lang dune 1.0)
```

Comment:
We should create these dune-project files in tests. This way we don't have to update all the tests at once when we'll change something.